.github/workflows/ci.yml

```
@@ -213,7 +234,7 @@ jobs:
         if: matrix.distro == 'debian' && matrix.version == 'bookworm'
         uses: actions/upload-artifact@v4
         with:
-          name: dangerzone-${{ matrix.distro }}-${{ matrix.version }}.deb
+          name: dangerzone-${{ matrix.distro }}-all.deb
```

```suggestion
          name: dangerzone-${{ matrix.distro }}-all.deb
```

Any particular reason why we want to change the original naming? I'm asking because in this context, the `debian` and `all` tags don't help much (the `all` is a bit misleading as well, since the original file has `amd64` in its name instead).

---

We can leave it just `dangerzone.deb`, if we want to indicate that it's suitable for every Debian version.

---

This is now fixed in the latest squashed commit of this PR